### ISSUE TITLE
cell span (colspan, table title) support

### ISSUE THREAD

**Issue by ORESoftware** [gajus/table#12]
Just looking for this simple feature in gajus/table

<img width="899" alt="screenshot 2016-04-09 18 03 19" src="https://cloud.githubusercontent.com/assets/11139560/14407422/6ef9b894-fe7d-11e5-994b-496567b75432.png">

Looking for somewhere in this library's output to put "Suman Test Results"

Does this lib support a table title?

**Comment by gajus** [comment]
No, there is no way no way make a certain cell span multiple columns. As far as I remember, this feature is not implemented on purpose: It would add a considerable complexity to otherwise simple codebase.

**Comment by gajus** [comment]
There is nothing that would stop you from using the low-level API (see [`./tests`](https://github.com/gajus/table/tree/7464b27999c5c6da43c754fb5f94931423d41839/tests)) to achieve the same result. Regardless, I would discourage such use as an unnecessary.

**Comment by gajus** [comment]
You can write a PR. I will consider to merge it. If the added complexity is within reason, I will merge it. Otherwise, I can support maintenance of a branch/ separate module that relies on the codebase or table and supports column spanning.

**Comment by ORESoftware** [comment]
Alrighty let me think about it!

**Comment by oncletom** [comment]
I guess it would have to imply some sampling if no column width defaults are provided.

```js
table([
  ["Test Results"],
  ["Metric", "Value", "Notes"],
  ["Suite name", "@Test1", ""],
  // ...
]);
```

The main difficulty is to understand which columns are merged into which ones. Without hint provided by the user it is difficult to know :-(

Unless by providing symbols?

```js
table([
  ["Test Results", ...table.merge.right(2)],
  ["Metric", "Value", "Notes"],
  ["Suite name", "@Test1", ""],
  // ...
]);
```

**Comment by nam-hle** [comment]
The feature has been included in v6.8.0
